accepted/future-releases/extension-types/feature-specification.md

```diff
@@ -701,6 +705,17 @@ Member Conflicts' occur as well in an extension type declaration.
 and has an instance member named `V`, and it is a compile-time error if it
 has a type parameter named `X` and it has an instance member named `X`.*
 
+If the representation name of an extension type is a private name `_n` then
+the representation variable is subject to promotion according to the rules
+about promotion of private instance variables, except that promotion is
+_not_ eliminated by the existence of other declarations (of any kind) named
+`_n` in the same library.
```

Comment:
If we had promotion of private _non-instance_, aka. statically resolved, variables, we should combine it into that instead of into "instance member access". But we don't. (Right?)

Reply:
Right, we do not currently promote any library or static variables.

> Do we need concepts to distinguish between _virtual_ instance members and _statically resolved_ instance members?

I usually mention the kind of enclosing declaration: "a class / mixin / mixin class / enum instance member" vs. "an extension / extension type instance member". It might be useful to generalize this, but I don't think it's a big deal.